lua/nordic/tests/init.lua

```diff
@@ -0,0 +1,51 @@
+local U = require('nordic.utils')
+
+M = {}
+
+local function pretty_string(value)
```

Comment:
I know you made this with love... but maybe `vim.inspect()`?

Reply:
That works 4bcea92